### PR TITLE
Add dnxcore50 support for SplitPackages

### PR DESCRIPTION
**Pull request by shirhatti**
@javiercn Can you have a look?

### CODE REVIEW

**Comment by javiercn** [comment]
@shirhatti Were you able to run the tool successfully on CoreCLR?

File: src/SplitPackages/project.json [hunk 1]
```diff
@@ -16,6 +16,11 @@
         "NuGet.Client": "3.4.0-*"
     },
     "frameworks": {
+        "dnxcore50": {
+            "dependencies": {
+                "NETStandard.Library": "1.0.0-rc2-*"
```
Comment (javiercn) [inline src/SplitPackages/project.json]: Just 1.0.0-*

**Comment by javiercn** [comment]
:shipit: After that

File: src/SplitPackages/project.json (hunk 1 above)
Comment (shirhatti) [inline src/SplitPackages/project.json]: @troydai ?

Comment (javiercn) [inline src/SplitPackages/project.json]: @pranavkm :smiley_cat:

Comment (troydai) [inline src/SplitPackages/project.json]: We mostly spent time on `dev` and `rel/1.0.0` branch where we use `rc2` packages only. More tests are done on `rc2` packages.

Comment (shirhatti) [inline src/SplitPackages/project.json]: So what's the consensus? `1.0.0-*` or `1.0.0-rc2-*`?

Comment (javiercn) [inline src/SplitPackages/project.json]: @shirhatti Whatever you prefer, we filter our packages to RC2, so the end result is going to be the same

Comment (shirhatti) [inline src/SplitPackages/project.json]: :+1:  I'm going to leave it at `1.0.0-rc2-*`

**Pull request closed without merging**